plugins/outputs/orionres/res/session/session.go

```diff
@@ -107,7 +107,17 @@ func (r *sessionResource) Delete(req *http.Request, id string) (int, interface{}
 }
 
 func (r *sessionResource) Update(req *http.Request, id string, value interface{}) (int, interface{}) {
-	return http.StatusNotFound, nil
+	s := value.(*Session)
```

Comment:
As written, a panic will occur if the type assertion fails.  Suggest returning `http.StatusBadRequest` if the type assertion fails.

Reply:
Fixed to return bad request and updated object